quickwit-search/src/search_stream/leaf.rs

```diff
@@ -38,8 +41,8 @@ use tokio_stream::wrappers::UnboundedReceiverStream;
 // to tonic::Status as tonic::Status is required by the stream result
 // signature defined by proto generated code.
 pub async fn leaf_search_stream(
-    query: Box<dyn Query>,
-    fast_field_collector_builder: FastFieldCollectorBuilder,
+    index_config: Box<dyn IndexConfig>,
+    request: &SearchStreamRequest,
     split_ids: Vec<String>,
     storage: Arc<dyn Storage>,
     output_format: OutputFormat,
```

Comment:
I think we can remove `output_format` param as it is in the `SearchStreamRequest`